### ISSUE TITLE
built in activity journal

### ISSUE THREAD

**Issue by danrabbit** [elementary/switchboard-plug-parental-controls#44]
We should have a built-in Zeitgeist activity journal page so parents can see which apps kids are spending time in.

**Comment by cassidyjames** [comment]
From duplicate #48:

>The new digital wellbeing features in iOS and Android both seem to track and display app usage, i.e. time spent in an app in their respective digital wellbeing dashboards. This seems like something that could be useful both for parents to understand their kids' usage, and also combined with #46 to enable personal digital wellbeing to better understand one's own usage.

**Comment by SixFingeredAmish** [comment]
I think if this feature is added, there should be a constant notification informing the user that they are monitored. Important to note that the user isn't always a child, could be a regular user being monitored. Just a good privacy practice I think.

A good implementation of this would be a persistent notification bar icon (say, an Eye icon), similar to when you have screen recording apps running. Upon click, there is a tooltip that says, "app and web usage is currently being monitored"

**Comment by cassidyjames** [comment]
@SixFingeredAmish I don't think we would track website usage (as that's a bit creepy), and if you set up monitoring for yourself, I am not entirely sure you need to be constantly reminded that you are monitoring yourself. 

If the situation is a non-admin account being monitored by an admin account, I can see some value to notifying the user. However, keep in mind it's a non-admin on an admin's computer, so there's all sorts of monitoring that could be happening anyway without the system or user knowing.

**Comment by SixFingeredAmish** [comment]
@cassidyjames Agreed! Even if it's not a catch-all, at least it's something

**Comment by cassidyjames** [comment]
Relevant art from Android:

Feature       | Screen                                                                                                               | Notes
------------- | -------------------------------------------------------------------------------------------------------------------- | -----
Dashboard     | ![Dashboard](https://user-images.githubusercontent.com/611168/54158289-c9105600-440f-11e9-9bfe-9ce7f2a00038.png)     | Shows pie chart of apps by screen time. Tapping an app opens its specific view of data. Tapping "Unlocks" or "Notifications" opens a sub-page with a chart and list of apps. Links to other related Digital Wellbeing settings are listed below.
Screen Time   | ![Screen Time](https://user-images.githubusercontent.com/611168/54158286-c9105600-440f-11e9-827d-77e03809da22.png)   | Tapping in the middle of the chart (on the amount of screen time) brings you here. The blue bar at the top also lets you switch between Screen Time, Unlocks, and Notifications. Shows the amount of screen-on time and the apps below with how long each was used. The :hourglass: icon lets you set an "app timer" where that app will be disabled once it has been used for that amount of time each day.
Unlocks       | ![Unlocks](https://user-images.githubusercontent.com/611168/54158287-c9105600-440f-11e9-853e-6f8cf28a0675.png)       | Dashboard sub-page that shows both the history of unlocking/waking up the device as well as the number of times apps are opened. The :hourglass: icon lets you set an "app timer" where that app will be disabled once it has been used for that amount of time each day.
Notifications | ![Notifications](https://user-images.githubusercontent.com/611168/54158288-c9105600-440f-11e9-8d47-a23817d234ed.png) | Dashboard sub-page that shows the number of notifications received, as well as the apps and how many they each sent. The :hourglass: icon lets you set an "app timer" where that app will be disabled once it has been used for that amount of time each day. This seems dumb and should be a link to the app's notification settings instrad.
App sub-page  | ![App](https://user-images.githubusercontent.com/611168/54158285-c9105600-440f-11e9-8219-6754915bdaa4.png)           | A confusingly-located but handy sub-page for apps. Tapping any app in the Dashboard or any of the Dashboard subpages brings you here. You get the same three sections for Screen Time, Opens, and Notifiations here with a graph specifically for this app. There's also a link to the App Timer and Notification controls for that specific app.

I really don't love how it's organized on Android (you can get lost in sub-pages and the hierarchy is not super clear), but the type of data here and some of the visualizations are useful to consider.
